src/main/java/soundchan/BotListener/BotListener.java

```diff
@@ -59,14 +64,27 @@ private void loadProperties(@NotNull Properties properties) {
         if(audioOnUserJoin) {
             String userAudioPath = properties.getProperty("userAudioFilePath");
             if(userAudioPath == null || userAudioPath.contentEquals("")) {
-                localManager = new LocalAudioManager(localFilePath, "usersound.properties");
+                userAudioPath = "usersound.properties";
             }
-            else {
-                localManager = new LocalAudioManager(localFilePath, userAudioPath);
+            localManager = new LocalAudioManager(localFilePath, userAudioPath);
+
```

Comment:
I see these two blocks are almost identical, probably could make it into a function with an optional argument

Reply:
That would be the blocks on line 71 and 81